include/AudioFileMP3.h

```diff
@@ -58,7 +58,7 @@ class AudioFileMP3 : public AudioFileDevice
 	}
 
 protected:
-	void writeBuffer(const SampleFrame* /* _buf*/, const fpp_t /*_frames*/) override;
+	void writeBuffer(const SampleFrame* /* _buf*/, const size_t /*_frames*/) override;
```

Comment:
The other audio files doesn't seem to have the commented description of arguments.